rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,172 @@
+### DCP PR:
+
+# RFC Process
+
+## Summary
+
+The Human Cell Atlas Data Coordination Platform (HCA DCP) evolved informal, consensus-driven processes to coordinate and document technical and governance decisions.
+
+This early *lightweight* and *egalitarian* approach enabled the DCP to achieve rapid velocity and incorporate a diverse set of innovative design concepts.
+
+As DCP continues to mature and scale, our community can more easily contribute and learn when there is a formal and consistent **Request for Comments (RFC)** process to propose enhancements to software or governance which are then maintained in a central repository.
```

Comment:
And here I'd say we're doing the RFC process to make our work process more "consensus-driven" and "egalitarian"....

Reply:
WFM.